### PR TITLE
VPN-3112: Auto start breadcrumb on Android

### PR DESCRIPTION
**Pull request by mcleinman**
## Description

Adding a row with a link to the help article for Android start on boot. (Users need to set this up from Android Settings themselves.)

While the Figma (attached to the ticket) for this has link icon center justified (along with the toggle), I made this top justified to match what the toggle actually looks like.

While there is a similar looking component on the help screen, it is not a row - and didn't work here. I also tried adjusting another component to make it work as a link row, but ultimately Lesley helped me realize that we needed to create a new component for this situation.

Screenshot 27

## Reference

VPN-3112

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [x] I have commented my code PARTICULARLY in hard to understand areas
- [x] I have added thorough tests where needed

### CODE REVIEW

File: nebula/ui/components/MZBottomSheet.qml [hunk 1]
```diff
@@ -17,7 +17,7 @@ import Mozilla.Shared 1.0
       contentItem: ColumnLayout {
           Rectangle {
               anchors.left: parent.left
-              anchors. right: parent.right
+              anchors.right: parent.right
```
Comment (mcleinman) [inline nebula/ui/components/MZBottomSheet.qml:20]: This is just documentation, so doesn't change anything. (If you expand this snippet you'll see it's a comment.) I noticed this typo, and fixed it.

File: nebula/ui/components/MZLinkRow.qml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import QtQuick.Layouts 1.14
+
+import Mozilla.Shared 1.0        
+        
+        
+RowLayout {
+    id: toggleRow
```
Comment (lesleyjanenorton) [inline nebula/ui/components/MZLinkRow.qml:12]: nit: linkRow

File: src/ui/screens/settings/ViewPreferences.qml [hunk 3]
```diff
@@ -31,6 +31,16 @@ MZViewBase {
...
         spacing: MZTheme.theme.windowMargin
 
+        MZLinkRow {
+            objectName: "androidStartAtBootLink"
+
+            accessibleName: _startAtBootTitle
+            title: _startAtBootTitle
+            subLabelText: MZI18n.SettingsStartAtBootSubtitle
+            visible: Qt.platform.os === "android"
+            destinationUrl: "https://support.mozilla.org/kb/how-enable-always-vpn-android"
```
Comment (lesleyjanenorton) [inline src/ui/screens/settings/ViewPreferences.qml:41]: Suggestion: For the sake of consistency with `MZExternalLinkListItem{}` and `MZToggleRow` we could
- move this sumo URL to constants.h and then register a new url label for it in mozillavpn.cpp
- change the onClicked handler on L73 of `MZLinkRow` to be `linkRow.clicked()` and add `signal clicked()` somewhere around L20 of `MZLinkRow` (check out MZToggleRow as an example)
- drop the `destinationUrl` property from `MZLinkRow` and then change this line to `onClicked: MZUrlOpener.openUrlLabel("theNewLabel")`

**Review by lesleyjanenorton — APPROVED** [review]
R+ with 1 non-blocking comment :)

File: nebula/ui/components/MZLinkRow.qml (hunk 2 above)
Comment (mcleinman) [reply nebula/ui/components/MZLinkRow.qml]: Thanks, fixed.

File: src/ui/screens/settings/ViewPreferences.qml (hunk 3 above)
Comment (mcleinman) [reply src/ui/screens/settings/ViewPreferences.qml]: Thanks, updated

**Pull request merged**